### PR TITLE
Complete requirements of weather app

### PR DESCRIPTION
**Pull request by julian3493**
### Complete the weather app with the next requirements:
 
- Use API to get weather data
- Use async / await to process de data of the API
- Use Webpack
- Use ES6 modules
- Add images that change depending on the weather

### CODE REVIEW

File: src/modules/apiData.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { setValues, getUnits } from './set_values';
+import addImg from './addImg';
+
+const getData = async (city) => {
+  const unit = getUnits();
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=${unit}&appid=70c8ad9d3ce4e9eba23caf221443c534`, { mode: 'cors' });
+  const data = await response.json();
+  const values = {};
+  values.city = data.name;
```
Comment (Forison) [inline src/modules/apiData.js]: **[OPTIONAL]**
* [x] Use object destructuring to avoid repetition. we can destructure `name, weather` etc, from data.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change required❌

## Hi @julian3493 

#### Good job on this one, your Pull Request and Readme are informing, your code is organized well into modules, good job also on making sure you stuck to the ES6 syntax as required. Here some other improvements we can do to make it even better😊.

* [x] It has error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
- Let us use error handling block when making API calls. `try and catch` is my favorite. You can use this as foundation to notify user when they enter a wrong city name, this will make more UI friendly.

* [x] Let us always capitalize our "I" and API. The latter is an abbreviation and the former is represents a you. 

**[OPTIONAL]**

* [ ] It would be better to notify the user when they enter a wrong city name, this will make more UI friendly.

* [ ] It fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location

* [ ] it has a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

* [ ] Make sure to always capitalize your commit messages

* [ ] Adding a description to your repository is a plus, since it enhance the professionality of the repository.
* [ ]  Packages like `axios` save you the trouble of always converting your response to `json`

#### That is it, let us make these few changes and submit a re-review🙂

#### Happy coding👨‍💻

Addo Forison

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
-# weather-app
\ No newline at end of file
+# weather-app
+> In this project I build a small page where you can check the weather of a city it also shows temperature, humidity and sensation of temperature, i get those values using an api and another diferent api to get random images. For this project I used webpack and ES6 modules
```
Comment (Forison) [inline README.md]: API and I should Uppercase

File: src/modules/apiData.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { setValues, getUnits } from './set_values';
+import addImg from './addImg';
+
+const getData = async (city) => {
+  const unit = getUnits();
```
Comment (Forison) [inline src/modules/apiData.js]: * [ ] Make API calls within error handling block

**Review by wathigo — APPROVED** [review]
### APPROVED :heavy_check_mark: 
Great job implementing the requested changes.

**You may move on.**

**Happy Coding :computer: 
Simon Wathigo**

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**